.github/ISSUE_TEMPLATE/30_validation_issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Validation issue
+about: My `CITATION.cff` file is not valid and I need help
+title: ''
+labels: 'help wanted', 'validation'
+assignees: ''
+
+---
+
+<!--
+Please have a look at the section on validating your CITATION.cff file before submitting this issue: https://github.com/citation-file-format/citation-file-format#validation-heavy_check_mark.
+-->
+
+<!-- Don't delete the following note! -->
+Please note that issues with the validity of single CITATION.cff files may take some time to be picked up by the Citation File Format maintainers themselves. Therefore, if you are reading this issue and know how to validate `CITATION.cff` files, please help out if you can!
+
+## Invalid `CITATION.cff` file
+
+```yaml
+Paste your invalid CITATION.cff file here!
+```
+
+## Context
+
+- Please describe what you want to do (create a valid `CITATION.cff` file, show the citation information in the sidebar of your GitHub repository, ...)
```

Comment:
```suggestion
- Please describe what you want to do (create a valid `CITATION.cff` file, show the citation information in the sidebar of your GitHub repository, ...)

```

Reply:
Weird, I have a blank line at the end in my local file ... Anyhow...